source/includes/api/_dataset.md

```diff
@@ -1053,21 +1291,32 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/:dataset_id/data-overwrite
 }'
 ```
 
-<aside class="notice">
-    This is an authenticated endpoint!
-</aside>
+Using this endpoint, you can add completely replace the data of an already existing dataset. All previously existing data will be permanently deleted. You can either provide the URL(s) for the file(s) containing the data you wish to add, or simply provide that data in the body of your request, as a JSON object. There are no requirements regarding similarity of the data structure between existing and new data - your overwrite data can have a completely different data schema.
```

Comment:
Why can't I do this in a normal PATCH operation?
What happens to the old file in S3?